docs/configuration/single-process-config-blocks-tls.yaml

```diff
@@ -0,0 +1,100 @@
+
```

Comment:
For each pre-cooked config we provide, we have an integration test in `integration/getting_started_single_process_config_test.go`. Would be great if you could add a test there for this file (a new test, which can be an existing one you copy, paste and modify as needed).